bats/tests/helpers/vm.bash

```diff
@@ -21,11 +32,32 @@ factory_reset() {
     fi
 }
 
+# Turn `rdctl start` arguments into `npm run dev` arguments
```

Comment:
We should consider making `npm run dev` take the same arguments as `rdctl start` instead. That's better for our sanity. :)

Reply:
That works for me too, but should be a separate PR.

Reply:
Yeah, definitely not as part of this PR.